### DISCUSSION TITLE
Codemagic does not re-fetch the codemagic.yaml config file

### DISCUSSION THREAD

**Discussion by radiKal07** [codemagic-ci-cd/codemagic-docs#1796] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Commit codemagic.yaml to master branch
2. Go to Codemagic and click refresh button

### Expected results

The new workflow appears

### Actual results

The new workflow does not appear

### Build id (optional)

App ID: 5da4c5f87338b04c6efa3ae8

**Comment by dtrdic** [comment]
Hello @tudor07 have you created builds before or this is something that only started happening? have configured the git integration with codemagic already? here are some guidance that you can check https://docs.codemagic.io/getting-started/adding-apps/#connecting-a-git-repository

**Comment by radiKal07** [comment]
It was an issue on my side
